modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -290,3 +287,4 @@ $ aws --output json logs describe-log-groups | jq .logGroups[].logGroupName
 ----
 
 The `groupBy` field affects the application log group only. It does not affect the `audit` and `infrastructure` log groups.
+////
```

Comment:
Any info in the lines above referencing 'groupBy` is outdated and incorrect in v6.    This example can probably be done away with completely.     
 For example, if you want UUID to show after the app namespaces, you would use `groupName: my-cluster-7977k.{.namespace_id||"unknown}` and place this in a pipeline for app logs only.